src/cli/CMakeLists.txt

```diff
@@ -53,4 +53,4 @@ if(MSVC_IDE)
 	set_source_files_properties(win32/precompiled.c COMPILE_FLAGS "/Ycprecompiled.h")
 endif()
 
-install(TARGETS git2_cli RUNTIME DESTINATION ${CMAKE_INSTALL_BINDIR})
+install(TARGETS git2_cli EXPORT ${LIBGIT2_TARGETS_EXPORT_NAME} RUNTIME DESTINATION ${CMAKE_INSTALL_BINDIR})
```

Comment:
Curious why we'd export the CLI targets? How would somebody make use of this and why would they want it? 🤔

Reply:
You are correct, for the git cli, the export is unnecessary.